sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -324,20 +324,6 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
       | FROM src LIMIT 1
     """.stripMargin)
 
-  createQueryTest("Date comparison test 2",
-    "SELECT CAST(CAST(0 AS timestamp) AS date) > CAST(0 AS timestamp) FROM src LIMIT 1")
```

Comment:
depends on the system time zone, it should be ...
- West of GMT `daysSinceEpoch` is `-1`
- East of GMT `daysSinceEpoch` is `0`
```
 "CAST(0 AS timestamp)" --> "1 Jan 1970 00:00:00 GMT"
 "CAST(0 AS timestamp)" --> "31 Dec 1969 16:00:00 PST"
```